app/src/components/integrations/containers/integrationSettingsContainer/integrationSettingsContainer.jsx

```diff
@@ -57,6 +59,7 @@ export class IntegrationSettingsContainer extends Component {
       enabled: true,
       integrationParameters: formData,
     };
+    const integrationParameters = omit(data.integrationParameters, OMIT_KEYS);
```

Comment:
Such `omit` already exists in the `src/controllers/plugins/sagas.js` `function* updateIntegration`.
To reuse this normalized `integrationParameters` make sense to provide integration data to the callback from the saga (line 127) and use it [here](https://github.com/reportportal/service-ui/pull/2618/files#diff-1b7acf2538fb238032aed222558b2f44ad5cb896602ad6bf286658e8cfb4908eR73).